### PR TITLE
Set addon services stateful by default on kontena app init command

### PR DESCRIPTION
**Pull request by nevalla**
This PR sets Heroku addon services stateful by default on `kontena app init` command.

### CODE REVIEW

**Comment by jakolehm** [comment]
What happens if user deploys another app to the same grid? Does it throw error because load balancer exists or something else?

**Comment by nevalla** [comment]
With this current implementation loadbalancer is app specific so if there is another loadbalancer service on the same grid, apparently the public port 80 is already reserved and deploy fails.

**Comment by nevalla** [comment]
I have removed load balancer at this moment and let's figure out proper way to do it later.

File: cli/lib/kontena/cli/apps/common.rb [hunk 1]
```diff
@@ -80,5 +80,51 @@ def dockerfile_exist?
...
+              'image' => 'postgres:latest',
+              'environment' => ["DATABASE_URL=postgres://#{prefix}postgres:@postgresql:5432/postgres"]
+          },
+          'mongolab' => {
+              'image' => 'mongo:latest',
+              'environment' => ["MONGOLAB_URI=#{prefix}mongolab:27017"]
+          },
+          'memcachedcloud' => {
+              'image' => 'memcached:latest',
+              'enviroment' => ["MEMCACHEDCLOUD_SERVERS=#{prefix}memcachedcloud:11211"]
```
Comment (jakolehm) [inline cli/lib/kontena/cli/apps/common.rb]: let's fix this typo

**Comment by nevalla** [comment]
ping @jakolehm

**Comment by jakolehm** [comment]
I think all the typos are gone.. so LGTM!

**Pull request merged**